Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+dynamic inner project network average.
+
+BOINC has been rewarding scientific work units in a credit system since
+2003. In case a project maintainer maliciously decides to give out more
+credits than appropriate, it only affects the inner project competition.
+The maximum share of new Gridcoins for this project stays the same
+percentage (subject to number of total projects). And projects can be
+removed by a blockchain vote.
+
+Security in Gridcoin is derived from Blackcoin's industry-leading
+Proof-of-Stake.
+
+# Mining (Research)
+
+## How do I start mining?
+
+First, [Get some coins](Get_some_coins "wikilink"), then see [Mining
+setup](Mining-Setup "wikilink").
+
+## My CPID is not in the neural network, What do I do?
+
+If you are in the pool, this is normal. If you are attempting to
+solo-stake, go through the following:
+
+If you have just begun your BOINC journey it is quite possible your work
+units haven't been verified by the necessary servers. Don't Panic,
+Gridcoin is a game of patience. Just relax, take a day or two, and see
+if the problem persists. If so, proceed to the following steps.
+
+To ensure your wallet is fully unlocked, lock your wallet with a
+passphrase (make it LONG, a password manager like KeePass is
+recommended) and then unlock it with "for staking only" UNticked.
+
+Next, go to Help -\> Debug Window, go to the Console tab and type
+`advertisebeacon` without the quotes.
+
+This will (if successful) send a small transaction at the cost of
+-0.00011 GRC. This should add your CPID with the next superblock.
+
+To see if your CPID is included, go to the Console and type `beaconreport`. If your CPID is present, sync the Neural Network by
+typing `syncdpor2` in the Console. If successful, the output
+should be '"syncing": 1'.
+
+If your CPID is not included, wait for the next superblock. In case your
+CPID isn't included after a few superblocks, try resending your beacon
+(`advertisebeacon`).
+
+Lock your wallet again, but this time unlock it with "for staking only"
+ticked.
+
+## How long does it take to start receiving Research (mining) credit?
+
+This depends on several factors. You must have some coins to begin with,
+some have started with as few as 100 coins, but the general recommendation now is around 5000. You must send a "beacon",
+this is how your information is populated into the Neural Network which
+controls payments, see [Mining setup](Mining-Setup "wikilink"). You must
+have over 100 RAC on at least 1 whitelisted BOINC project before you can
+send a beacon to receive Research credit. After your CPID is in the
+Neural Network, all research credits you earn will accrue in a "savings
+account" for a rolling 14 day period. You have up to 2 full weeks to
+stake a block without losing any research credit
+earned.
+
+## Is it true that someone would not lose reward if blocks are created on a fork?
+
+YES! If you find a block on a fork your "owed" balance will be reduced

```

Comment:
This section should probably just be rewritten. It can be made much simpler to something like:

```If you stake on a fork, the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear since they shouldn't have shown up in the first place. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again. ```